doc/source/user/basics.creation.rst

```diff
@@ -76,7 +76,8 @@ the computation, here ``uint32`` and ``int32`` can both be represented in
 as ``int64``. 
 
 The default NumPy behavior is to create arrays in either 64-bit signed
-integers or double precision floating point numbers, ``int64`` and
+integers (32-bit signed integers on Windows platform) or double precision
+floating point numbers, ``int64`` (``int32`` on Windows platform) and
```

Comment:
What does 32-bit Linux use?

Reply:
Thanks for pointing this out @bashtage, it is indeed determined by the datamodel for C long type. Just checked on linux 32bit machine and indeed it becomes an int32 type, so i think we can only leave it at "platform defined" like how its in (https://numpy.org/doc/stable/user/basics.types.html). So, basically in this doc, a clarification that np.array will by default create an array with elements of type int32 or int64 depending on the platform.

Reply:
It might be worth nothing, since it is surprising, that the default int on 64 bit windows is 32-bits wide. I think in most other cases it behaves as expected. (There are strange implementations where ilp32 is used on ARM64, but I don't know if this is remotely common)